src/hotspot/share/runtime/os.hpp

```diff
@@ -326,7 +327,8 @@ class os: AllStatic {
 
   static int    vm_allocation_granularity();
   static char*  reserve_memory(size_t bytes, char* addr = 0,
-                               size_t alignment_hint = 0, int file_desc = -1);
+                               size_t alignment_hint = 0, int file_desc = -1,
+                               bool executable = false);
```

Comment:
Please make this default=false too, and revert the hunks where this is explicitly set to false